src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/request/UpdateReportRequest.kt

```diff
@@ -0,0 +1,61 @@
+package uk.gov.justice.digital.hmpps.incidentreporting.dto.request
+
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.ValidationException
+import jakarta.validation.constraints.Size
+import uk.gov.justice.digital.hmpps.incidentreporting.jpa.Report
+import java.time.Clock
+import java.time.LocalDateTime
+
+@Schema(description = "Payload to update key properties of an incident report")
+data class UpdateReportRequest(
+  @Schema(description = "When the incident took place", required = false, defaultValue = "null", example = "2024-04-29T12:34:56.789012")
+  val incidentDateAndTime: LocalDateTime? = null,
+  @Schema(description = "The NOMIS id of the prison where incident took place", required = false, defaultValue = "null", example = "MDI")
+  @field:Size(min = 2, max = 6)
+  val prisonId: String? = null,
+  @Schema(description = "Brief title describing the incident", required = false, defaultValue = "null")
+  @field:Size(min = 10, max = 255)
+  val title: String? = null,
+  @Schema(description = "Longer summary of the incident", required = false, defaultValue = "null")
+  val description: String? = null,
+  @Schema(description = "Username of person who created the incident report", required = false, defaultValue = "null")
+  @field:Size(min = 3, max = 120)
+  val reportedBy: String? = null,
+  @Schema(description = "When the incident report was created", required = false, defaultValue = "null", example = "2024-04-29T12:34:56.789012")
+  val reportedAt: LocalDateTime? = null,
+
+  @Schema(description = "Whether the parent event should also be updated", required = false, defaultValue = "false", example = "true")
+  val updateEvent: Boolean = false,
```

Comment:
report deletion already has an option to propagate to the parent event, so it makes sense to allow propagating field updates. there are no other fields in the event than those already listed here – we might not need any special event-editing endpoints!

(of course, the parent event is very likely to change shape and be renamed in future)